src/lib/enhancers/__tests__/createShowMore-test.ts

```diff
@@ -0,0 +1,54 @@
+import { createShowMore } from '../createShowMore';
+
+describe('createShowMore', () => {
+  test('goes through the lifecycle', () => {
```

Comment:
what's meant by this test name?

Reply:
It tests that it goes through the subscription lifecycle of the enhancer as expected. We can split this test once we determine the final API.